inlong-dashboard/src/ui/pages/GroupDetail/ResourceInfo/index.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect, useMemo, forwardRef, useState } from 'react';
+import { Divider, Table } from 'antd';
+import FormGenerator, { useForm } from '@/ui/components/FormGenerator';
+import { useRequest } from '@/ui/hooks';
+import { useTranslation } from 'react-i18next';
+import { CommonInterface } from '../common';
+
+type Props = CommonInterface;
+
+const Comp = ({ inlongGroupId, isCreate }: Props, ref) => {
+  const { t } = useTranslation();
+
+  const [form] = useForm();
+
+  const title = [
+    {
+      label: 'Agent',
+      value: 'AGENT',
+    },
+    {
+      label: 'DataProxy',
+      value: 'DATAPROXY',
+    },
+    {
+      label: 'Pulsar',
+      value: 'PULSAR',
+    },
+    {
+      label: 'TubeMQ',
+      value: 'TUBEMQ',
+    },
+    {
+      label: 'SortZookeeper',
+      value: 'ZOOKEEPER',
+    },
+    {
+      label: 'DbsyncZk',
+      value: 'DBSYNC_ZK',
+    },
+    {
+      label: 'SortThive',
+      value: 'SORT_THIVE',
+    },
+    {
+      label: 'SortCK',
+      value: 'SORT_CK',
```

Comment:
Where are these cluster types used?

Reply:
> Where are these cluster types used?

used in this place
![image](https://github.com/apache/inlong/assets/88174078/b85aa113-21cf-4e69-b9d6-7a089ef74946)